other/upgrade_2-1_postgresql.sql

```diff
@@ -2055,15 +2055,19 @@ CREATE INDEX {$db_prefix}ban_items_id_ban_ip ON {$db_prefix}ban_items (ip_low,ip
 --- helper function for ip convert
 /******************************************************************************/
 ---# the function migrate_inet
-CREATE OR REPLACE FUNCTION migrate_inet(val IN anyelement) RETURNS inet
-AS
-$$
-BEGIN
-   RETURN (trim(val))::inet;
-EXCEPTION
-   WHEN OTHERS THEN RETURN NULL;
-END;
-$$ LANGUAGE plpgsql;
+---{
+upgrade_query("
+	CREATE OR REPLACE FUNCTION migrate_inet(val IN anyelement) RETURNS inet
+	AS
+	$$
+	BEGIN
+	   RETURN (trim(val))::inet;
+	EXCEPTION
+	   WHEN OTHERS THEN RETURN NULL;
+	END;
+	$$ LANGUAGE plpgsql;"
+);
+---}
```

Comment:
Albert you're a hero! :)

Got my board up and running. The problem with the plpgsql function actually was that I pasted the new one on top of the old and then forget to remove the old... So pilot error. Board is up and running, update all done.

I'll do a fresh update from existing data and then let our forum users loose on the beta install :)

Thanks a lot! -Reko